src/locales/en.json

```diff
@@ -376,6 +376,8 @@
     "show": "Show results"
   },
   "browse-page": {
+    "audio-result-count": "no audios | {localeCount} audio | {localeCount} audios",
```

Comment:
I was under the impression that "audio", being uncountable, could not be pluralised this that and thought that maybe "audio files" or "audio clips" would be better.

I was so wrong! This is awesome.

Reply:
I had the same back-and-forth :) Thank you for sharing!